meta-zarhus-webkit/recipes-networking/nginx/nginx/default_server.site

```diff
@@ -0,0 +1,13 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
```

Comment:
Will this allow connections from outside? Maybe we don't want that by default (e.g. security)?
Maybe something like this would work to only listen on localhost

```suggestion
    listen 127.0.0.1 default_server;
```

Reply:
My bad, fixed here https://github.com/zarhus/meta-zarhus/compare/0d427b019fc30c5631105f0a549970d953ee01fe..99fb8646324027dfbb120d6f77d4cc48ea5895cd.